docs/stable/1-client.md

```diff
@@ -94,20 +96,24 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
+On top of this we run the RLPx-based subprotocol [Waku
+V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md) for privacy-preserving messaging.
 
 There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+implements Waku V0 specification.
 
 #### Node discovery and roles
 
 There are four types of node roles:
 1. Bootstrap nodes
-2. Whisper relayers
+2. Waku relayers
 3. Mailservers (servers and clients)
 4. Mobile nodes (Status Clients)
 
+```js
+// TODO do waku relayers exist? Are they different from mobile nodes?
```

Comment:
Should be essentially same as for Whisper. It'd be good if we kept Whisper in history here, so we have Whisper and Waku relayers. Then, at a later stage, we can deprecate Whisper completely.

cc @jakubgs re Waku relays in cluster